### PR TITLE
141 integrate put kebiez

### PR DESCRIPTION
**Pull request by Kebiez**
#141 Integrate put request for updating data

@GabrielAdward 
@GLADOSv2 

## Task list 
- [x] Checked the code structure so it follows guide lines
- [x] Checked that all the points in the Issue are solved
- [x] Checked that all the old tests still got approved

## Added tests
- [ ] Yes, I have added tests
- [x] No, I have not added tests
- [ ] No, I need help with adding tests

### CODE REVIEW

**Comment by HeWhoThreadsLightly** [comment]
Have you merged in the latast changes from development?

**Review by HeWhoThreadsLightly — APPROVED** [review]
Forme validation needs to be fixed so it is in sync with backend.

File: src/components/profileTab/UpdateProfileInformation.tsx [hunk 1]
```diff
@@ -56,6 +64,31 @@ const UpdateProfileInformation = (props: any) => {
   const classes = useStyles();
   const { UpdateUserProfile, handleInputValue, errors, msg } = ValidationForm();
 
+  async function fetchNewInfoFromTextArea(){
+    
+    let listOfNewProfileInfo = [];
+    listOfNewProfileInfo.push(document.getElementById("index " + 0 + " of textareas") as HTMLInputElement);
```
[the remark this answers is not in the document]
Comment (HeWhoThreadsLightly) [reply src/components/profileTab/UpdateProfileInformation.tsx:70]: Those issues are for the next year, if you would kindly write them for next year.

[the remark this answers is not in the document]
Comment (Kebiez) [reply src/components/profileTab/UpdateProfileInformation.tsx:70]: Yeah I know, I tried using a for loop but did not work for some weird reason.

**Review by GabrielAdward — APPROVED** [review]
LGTM!

**Pull request merged**